beheer.md

```diff
@@ -23,3 +24,65 @@ Bij de overdracht van een standaard van ontwikkeling naar beheer moeten ook de n
 Aangezien bij overdracht van ontwikkeling naar beheer de GitHub repository as-is wordt overgenomen kan de backlog zelf blijven waar deze is.
 
 Overdracht van de backlog en GitHub repository is een officieel moment en moet plaatsvinden voordat er sprake kan zijn van in beheername van een Open API
+
+## Werken met Docker
+
+Zie https://docs.docker.com/get-started/ en specifiek https://docs.docker.com/get-started/part2/ en https://docs.docker.com/get-started/part3/ voor een diepere uitleg over Docker.
+
+Voor het vervaardigen van Docker images heb je het volgende nodig:
+
+* een goed werkende Docker installatie;
+* een folder met daarin:
+  - een Dockerfile (met de naam 'Dockerfile');
+  - een requirements file (met de naam requirements.txt);
+  - een python applicatie;
+  - evt. een 'docker-compose.yml' file.
```

Comment:
Ter info: Simpelgezegd brengt een compose file meerdere Docker containers samen. Typisch voorbeeld is een database en een applicatie, elk hun eigen Docker.